src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -110,13 +119,13 @@
                     EMPTY_PHONE, new Email("olivealice@example.com"),
                     new Address("Blk 11 Ang Mo Kio Street 74, #11-04"),
                     new School("NYJC"), EMPTY_ACAD_STREAM, EMPTY_ACAD_LEVEL,
-                    EMPTY_REMARK, getTagSet("zoom"),
+                    EMPTY_REMARK, getTagSet("J2"),
                     getLessonSet(getSampleLessons()[3])),
             new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
                     new Phone("91009182"), EMPTY_EMAIL,
                     new Address("Blk 436 Serangoon Gardens Street 26, #16-43"),
                     EMPTY_SCHOOL, EMPTY_ACAD_STREAM, EMPTY_ACAD_LEVEL,
-                    new Remark("Weak at trigo"), getTagSet("zoom"),
+                    new Remark("Weak at trigo"), getTagSet("SEC2", "OLVL"),
```

Comment:
not sure if it's a good idea to put the acad level as a tag. And anyway why sec 2 taking o level hahaha

Reply:
@lingshanng i think its fine thoooo

Reply:
I agree with Lingshan. Use the acad level field for acad level at least on our side. If the user wants to do this, then that's up to them.

Reply:
eh why need change sample data ;-; if this changes, I need redo all the annotations for ug screenshots again

Reply:
Maybe make disclaimer that tags may differ if don't want to change? I think some differences in sample data probably not considered a bug